doc/semieunit.xml

```diff
@@ -0,0 +1,292 @@
+############################################################################
+##
+#W  semieunit.xml
+#Y  Copyright (C) 2016                                   Christopher Russell
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+##
+
+<#GAPDoc Label="IsMcAlisterTripleSemigroup">
+  <ManSection>
+    <Prop Name = "IsMcAlisterTripleSemigroup" Arg = "S"/>
+    <Returns><K>true</K> or <K>false</K>.</Returns>
+    <Description>
+      A <E>McAlister triple semigroup</E> is an E-unitary inverse semigroup
```

Comment:
You should talk with James already about how to organise this documentation. My intuition would be to have this part of the doc explain literally what filter `IsMcAlisterTripleSemigroup` describes (technically), and for all this mathematical info to be in the documentation for the creation functions.